source/jormungandr/jormungandr/parking_space_availability/car/common_car_park_provider.py

```diff
@@ -73,9 +83,32 @@ def get_informations(self, poi):
         if data:
             return self.process_data(data, poi)
 
-    def support_poi(self, poi):
+    def _is_poi_coords_within_shape(self, poi):
```

Comment:
Not sure: this logic may be shared with BSS also, so moved one class up.

Reply:
Moved one class up !